src/test/java/co/org/osso/api/ApliApplicationCalculatorTests.java

```diff
@@ -0,0 +1,60 @@
+package co.org.osso.api;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApliApplicationCalculatorTests {
+
+    CalculatorService CalculatorServiceTarget = new CalculatorService();
+
+    @Test
+    public void checkCalculatorServiceFibonacci(){
+        List<Integer> fib = new ArrayList<>();
+        fib.add(1);
+        fib.add(1);
+        fib.add(2);
+        fib.add(3);
+        fib.add(5);
+        Assert.assertEquals(fib, CalculatorServiceTarget.getFibonacci(5));
```

Comment:
Be careful of doing `assertEquals` on collections (maps, lists, sets etc.), as these could give you some unexpected results sometimes. It's often better to loop over them and assert that each element is equal for the same index.